4-projects/business_unit_1/shared/example_infra_pipeline.tf

```diff
@@ -15,7 +15,20 @@
  */
 
 locals {
-  repo_names = ["bu1-example-app"]
+  use_csr = var.cloudbuildv2_repository_config.repo_type == "CSR"
+  csr_repo_config = {
+    repo_type = "CSR"
+    repositories = tomap({
+      "bu1-example-app" = {
+        repository_name = "bu1-example-app"
+        repository_url  = ""
+      }
+    })
+    github_pat                        = null
+    github_app_id                     = null
+    gitlab_read_authorizer_credential = null
```

Comment:
why are the credentials set in locals here?
Other tf files that reference this use `var.cloudbuildv2_repository_config`

Reply:
this is being used on `modules/infra_pipelines` input.

the value is used to replicate the same behavior as today (CSR with bu1-example-app being created) by default

if the user specified something different than CSR, the local value will not be used:

```hcl
module "infra_pipelines" {
  source = "../../modules/infra_pipelines"
  count  = local.enable_cloudbuild_deploy ? 1 : 0

  org_id                         = local.org_id
  cloudbuild_project_id          = module.app_infra_cloudbuild_project[0].project_id
  cloud_builder_artifact_repo    = local.cloud_builder_artifact_repo
  remote_tfstate_bucket          = local.projects_remote_bucket_tfstate
  billing_account                = local.billing_account
  default_region                 = var.default_region
  cloudbuildv2_repository_config = local.use_csr ? local.csr_repo_config : var.cloudbuildv2_repository_config
  private_worker_pool_id         = local.cloud_build_private_worker_pool_id
}
```